src/main/java/com/esri/samples/scene/terrain_exaggeration/TerrainExaggerationController.java

```diff
@@ -44,18 +45,19 @@ public void initialize() {
       sceneView.setArcGISScene(scene);
 
       // add base surface for elevation data
-      Surface surface = new Surface();
+      surface = new Surface();
       final String elevationImageService =
-          "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer";
+              "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer";
       surface.getElevationSources().add(new ArcGISTiledElevationSource(elevationImageService));
       scene.setBaseSurface(surface);
 
       // set exaggeration of surface to the value the user selected
       exaggerationSlider.valueChangingProperty().addListener(o -> {

```

Comment:
I don't think this listener is necessary anymore with the event listener you added.